english/triathlon/triathlon.md

```diff
@@ -0,0 +1,8 @@
+You were always intrigued my endurance sports. 
```

Comment:
Not sure how to approach this. I did create a triathlon directory and then added the triathlon.mb file to it in my branch /english/triahtlon/triathlon.md